lib/ui/components/captioned_text_widget.dart

```diff
@@ -0,0 +1,67 @@
+// leading icon can be passed without specifing size, this component sets size to 20x20
```

Comment:
review the comment..? Also, this doesn't seems to be the right place to put it :)

Reply:
Oh yes, have fixed this